linera-storage/src/db_storage.rs

```diff
@@ -390,6 +400,24 @@ where
         Ok(missing_blobs)
     }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +385 to +401]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Like @afck mentioned, `missing_blob_states` should be semantically equivalent to this one (we'll only have a state in storage if we have a blob also). So we can probably just use this existing one instead and delete `missing_blob_states`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2894?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Ok, thank you.